hibernate-reactive-core/src/main/java/org/hibernate/reactive/loader/entity/impl/ReactivePlanEntityLoader.java

```diff
@@ -236,4 +257,173 @@ public String preprocessSQL(String sql,
 		//TODO!!!
 		return sql;
 	}
+
+	private static class ReactiveLoadPlanBasedResultSetProcessor extends ResultSetProcessorImpl
```

Comment:
Same question here: does the logic herein correspond to something specific in core?

Reply:
I've added Javadoc about this.